translations/override-translations/obdach.json

```diff
@@ -0,0 +1,7 @@
+{
+  "categories": {
+    "de": {
+      "metaDescription": "{{appName}} - die mehrsprachige Plattform, die wohnungslose und von Wohnungslosigkeit bedrohte Menschen mit wichtigen Anlaufstellen wie Beratungen und Notunterkünften verbindet."
+    }
+  }
+}
```

Comment:
Man, that description sounds like a lawyer wrote it but I guess that's not up to us to decide...

Reply:
ugh, don't mess with @FritjofK :D

Reply:
Good point, I don't really have a better suggestion either :D